DevExtreme.AspNet.TagHelpers.Generator/ChartSeriesSpecificSettings/Entry.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace DevExtreme.AspNet.TagHelpers.Generator.ChartSeriesSpecificSettings {
```

Comment:
Unconvincing. `Xml.Linq` worked perfectly for "PropOf". Using multiple approaches and adding extra references is fine.

P.S. You may want to add more details to this PR, why `XDocument` dependency is bad in the first place, and why it is to be removed.